pkgs/wireplumber/default.nix

```diff
@@ -8,4 +8,7 @@ wireplumber'.overrideAttrs(_: rec {
     rev = "refs/tags/${version}";
     hash = "sha256-+z7BQlRu8XYxE5vEFCfentdrQNuCP0RnQHiPx8/Yfl8=";
   };
+
+  # HACK: override postFixup from Wireplumber 0.5 PR
+  postFixup = null;
```

Comment:
This is my personal hack for https://github.com/NixOS/nixpkgs/pull/278760, I can probably get rid of it in my own config, but eh?